spec/lib/interest_days/calculation/icma_act_act_spec.rb

```diff
@@ -0,0 +1,128 @@
+# frozen_string_literal: true
+
+RSpec.describe InterestDays::Calculation::IcmaActAct do # rubocop:disable Metrics/BlockLength
+  describe "#day_count_factor" do # rubocop:disable Metrics/BlockLength
+    subject(:calc) { described_class.new(start_date: start_date, end_date: end_date) }
+
+    context "when start date in leap year" do
+      let(:end_date) { Date.new(2021, 2, 3) }
+
+      context "when 29th of february not in period" do
+        let(:start_date) { Date.new(2020, 5, 3) }
+
+        it "calculate the right day count factor" do
+          expectation = (end_date - start_date).fdiv(365)
+
+          expect(calc.day_count_factor).to eql(expectation)
+        end
+      end
+
+      context "when 29th of february is in period" do
+        let(:start_date) { Date.new(2020, 2, 3) }
+
+        it "calculate the right day count factor" do
+          expectation = (end_date - start_date).fdiv(366)
+
+          expect(calc.day_count_factor).to eql(expectation)
+        end
+      end
+    end
+
+    context "when end_date in leap year" do
+      let(:start_date) { Date.new(2019, 6, 1) }
+
+      context "when 29th of february not in period" do
+        let(:end_date) { Date.new(2020, 2, 3) }
+
+        it "calculate the right day count factor" do
+          expectation = (end_date - start_date).fdiv(365)
+
+          expect(calc.day_count_factor).to eql(expectation)
+        end
+      end
+
+      context "when 29th of february is in period" do
+        let(:end_date) { Date.new(2020, 5, 3) }
+
+        it "calculate the right day count factor" do
+          expectation = (end_date - start_date).fdiv(366)
```

Comment:
please be consistent between '366' and 'described_class::DAYS_IN_LEAP_YEAR' :)

Reply:
done!